source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -389,14 +389,18 @@ export default class WalletSendForm extends Component<Props, State> {
         });
       }
     } catch (error) {
+      let errorMessage = error;
```

Comment:
@tomothespian are you sure this part is right?

Reply:
I tested this one and that was only logic solution. This part is related to https://github.com/input-output-hk/daedalus/blob/369e9a9a41d475bb5f3bcc71f545425ba135af48/source/renderer/app/components/wallet/WalletSendForm.js#L393

and since message is introduced in component and used there we need to know that, read and pass to `this.context.intl.formatMessage(errorMessage)`

Reply:
Well, I think we should improve this as it is hard to understand what is going on here at the first look :(

Reply:
I agree ;)